installation/resources/installer-overrides-compass-gateway.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   namespace: kyma-installer
   labels:
     installer: overrides
-    component: compass
     kyma-project.io/installation: ""
 data:
   gateway.gateway.enabled: "true"
```

Comment:
Hmm, it would be good to move it to compass-specific override

Reply:
Ok. I created another PR for that: https://github.com/kyma-incubator/compass/pull/338